packages/api-gateway/__tests__/src/drivers/http/routes/chats/create-message.test.js

```diff
@@ -0,0 +1,148 @@
+// External dependencies
+const { faker } = require('@faker-js/faker');
+const { createMessage, isChatRelatedToUser } = require('@booking-services/messages');
+
+// Internal dependencies
+const { fastify } = require('../../../../../../src/drivers/http/server');
+
+// Mocks
+const mockMessageText = faker.lorem.sentence();
+const mockUserId = faker.database.mongodbObjectId();
+const mockChatId = faker.database.mongodbObjectId();
+
+const mockChat = {
+  _id: faker.database.mongodbObjectId(),
+  chatId: mockChatId,
+  text: mockMessageText,
+  createdBy: mockUserId,
```

Comment:
Don't you need deletedAt field?

Reply:
Yeah, we don't need it, I'll remove it in my next PR